dockerscripts/healthcheck.sh

```diff
@@ -19,7 +19,7 @@ set -x
 
 _init () {
     scheme="http://"
-    address="$(netstat -nplt 2>/dev/null | awk ' /(.*\/minio)/ { gsub(":::","127.0.0.1:",$4); print $4}')"
+    address="$(netstat -nplt 2>/dev/null | awk ' /LISTEN/ { gsub(":::","127.0.0.1:",$4); print $4}')"
```

Comment:
We have to make sure that the listener is only one here @nitisht i.e get the first element from the list.

Reply:
good point, I assumed there will be only one listener in this container, but we can get the first row only instead

Reply:
@harshavardhana PTAL